### PR TITLE
adding option for mapping prediction output to incident fields

### PR DESCRIPTION
**Pull request by lperryp**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/23062

## Description
adding option for mapping prediction output to incident fields 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

File: Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 1]
```diff
@@ -84,13 +86,21 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
...
     explain_result_hr = dict()
     explain_result_hr['TextTokensHighlighted'] = highlighted_text_markdown
-    explain_result_hr['Label'] = explain_result["Label"]
-    explain_result_hr['Probability'] = "%.2f" % explain_result["Probability"]
+    explain_result_hr['Label'] = predicted_label
+    explain_result_hr['Probability'] = "%.2f" % predicted_prob
     explain_result_hr['PositiveWords'] = ", ".join(positive_words)
     explain_result_hr['NegativeWords'] = ", ".join(negative_words)
+    if set_incidents_fields:
+        arg = {'customFields': {'DBotPrediction': predicted_label,
```
Comment (erezh31) [inline Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:98]: change to use OOB fields

File: Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 2]
```diff
@@ -84,13 +86,21 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
...
+                                'DBotPredictionProbability': predicted_prob,
+                                'DBotTextSuggestionHighlighted': highlighted_text_markdown}}
+        if original_incident_id is not None:
```
Comment (erezh31) [inline Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:101]: Do  not need this part

File: Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 3]
```diff
@@ -84,13 +86,21 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
...
+            arg['id'] = original_incident_id
+        demisto.executeCommand("setIncident", arg)
```
Comment (erezh31) [inline Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:103]: setIncident will perform on the current incident. just validate it's not playground

**Comment by kirbles19** [comment]
@lperryp Done

**Pull request merged**